pages/project-detail/index.vue

```diff
@@ -1,25 +1,222 @@
 <template>
   <div>
-    <project-single-header />
-    <project-single />
+    <project-single-header
+      :title="project.title"
+      :category="project.category"
+      :submit="project.submit"
+      :logo="project.logo"
+    />
+
+    <div class="bg-grey pt-8 pb-1">
+      <project-single-details
+        :title="project.title"
+        :description="project.description"
+        :team="project.team"
+        :cover="project.cover"
+        :socials="project.socials"
+      />
+
+      <project-single-dao-proposal
+        :round="project.proposal.round"
+        :requested="project.proposal.requested"
+        :wallet="project.proposal.wallet"
+        :rounds="project.proposal.rounds"
+      />
+    </div>
+
+    <project-single-roi-strategy
+      :description="project.roi.description"
+      :bang="project.roi.bang"
+      :buck="project.roi.buck"
+      :chance="project.roi.chance"
+      :calculation="project.roi.calculation"
+    />
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
-
-import ProjectSingleHeader from '~/components/app/project-detail/ProjectSingleHeader.vue'
-import ProjectSingle from '~/components/app/project-detail/ProjectSingle.vue'
+import EnumSocial from '@/components/enums/EnumSocial'
+import ProjectSingleDaoProposal from '@/components/app/project-detail/ProjectSingleDaoProposal.vue'
+import ProjectSingleDetails from '@/components/app/project-detail/ProjectSingleDetails.vue'
+import ProjectSingleRoiStrategy from '@/components/app/project-detail/ProjectSingleRoiStrategy.vue'
+import ProjectSingleHeader from '@/components/app/project-detail/ProjectSingleHeader.vue'
 
 export default Vue.extend({
+  name: 'ProjectDetail',
 
-  name: "ProjectDetail",
   components: {
-    ProjectSingle,
-    ProjectSingleHeader
+    ProjectSingleHeader,
+    ProjectSingleDetails,
+    ProjectSingleDaoProposal,
+    ProjectSingleRoiStrategy,
+  },
+
+  data() {
+    return {
+      project: {
```

Comment:
chore (blocking): looks like test data. Extract to an external file so that the code stays clean. You can even use the same structure as you would with calling an API